src/main/java/core/basesyntax/impl/Pair.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.impl;
+
+public class Pair<K, T> {
+    private int index;
+    private final K first;
+    private final T second;
```

Comment:
```suggestion
    private K key;
    private T value;
```
will you be able to create a setter for `value` if it is final?